fhir-persistence-jdbc/src/test/java/com/ibm/fhir/persistence/jdbc/test/util/QuantityParmBehaviorUtilTest.java

```diff
@@ -0,0 +1,701 @@
+    private static final Level LOG_LEVEL = Level.INFO;
+
+    //---------------------------------------------------------------------------------------------------------
+    // Supporting Methods: 
+    @BeforeClass
+    public static void before() throws FHIRException {
+        FHIRRequestContext.get().setTenantId("quantity");
+    }
+
+    @AfterClass
+    public static void after() throws FHIRException {
+        FHIRRequestContext.get().setTenantId("default");
+    }
+
+    private ParameterValue generateParameterValue(String value, SearchConstants.Prefix prefix) {
+        ParameterValue parameterValue = new ParameterValue();
+        parameterValue.setPrefix(prefix);
+        parameterValue.setValueNumber(new BigDecimal(value));
+        parameterValue.setValueCode("code");
+        parameterValue.setValueSystem("system");
+        return parameterValue;
+    }
+
+    private Parameter generateParameter(SearchConstants.Prefix prefix, SearchConstants.Modifier modifier,
+            String code, String... values) {
+        Parameter parameter = new Parameter(SearchConstants.Type.QUANTITY, code, modifier, null);
+        for (String value : values) {
+            parameter.getValues().add(generateParameterValue(value, prefix));
+        }
+        return parameter;
+    }
+
+    private Parameter generateParameter(SearchConstants.Prefix prefix, SearchConstants.Modifier modifier,
+            String value) {
+        return generateParameter(prefix, modifier, "Quantity", new String[] { value });
+    }
+
+    private Parameter generateParameter(SearchConstants.Prefix prefix, SearchConstants.Modifier modifier,
+            String... values) {
+        return generateParameter(prefix, modifier, "Quantity", values);
+    }
+
+    private void runTest(Parameter queryParm, List<Object> expectedBindVariables, String expectedSql, boolean sendNull)
+            throws Exception {
+        runTest(queryParm, expectedBindVariables, expectedSql, "Basic", sendNull);
+    }
+
+    public void runSignificantDigitsTest(String value, int significantDigits) {
+        assertEquals(NumberParmBehaviorUtil.calculateSignificantFigures(new BigDecimal(value)), significantDigits);
+    }
+
+    private void runTest(Parameter queryParm, List<Object> expectedBindVariables, String expectedSql,
+            String tableAlias, boolean sendNull)
+            throws Exception {
+        StringBuilder actualWhereClauseSegment = new StringBuilder();
+        List<Object> actualBindVariables = new ArrayList<>();
+
+        QuantityParmBehaviorUtil behavior = new QuantityParmBehaviorUtil();
+        behavior.executeBehavior(actualWhereClauseSegment, queryParm, actualBindVariables,
+                tableAlias, generateDao(sendNull));
+
+        if (log.isLoggable(LOG_LEVEL)) {
+            log.info("whereClauseSegment -> " + actualWhereClauseSegment.toString());
+            log.info("bind variables -> " + actualBindVariables);
+        }
```

Comment:
i havn't seen this pattern used in our codebase for INFO-level messages...are you sure we need to guard these?

Reply:
Fixed